d2l-rubric-editable-score.js

```diff
@@ -304,12 +345,16 @@ Polymer({
 	},
 
 	_scoreChanged: function() {
-
 		this.overriddenStyling = this._shouldShowOverrideState(this.scoreOverridden, this.readOnly, this.hovered);
 
 		if (this.scoreOverridden && this.hovered === -1 && !this.totalScore) {
 			this.score = this.overriddenScore;
 		}
+
+		this.scoreWidth = this.score ? this.score.toString().length : 1;
+		this.scoreFractional = this.score ?
+			Number(this.score) === this.score && this.score % 1 !== 0 :
+			false;
```

Comment:
super small nit, but.....
```suggestion
		this.scoreFractional = typeof this.score === "number" && this.score % 1 !== 0;
```